assisted-iso-create/main.go

```diff
@@ -0,0 +1,102 @@
+package main
+
+import (
+	"bytes"
+	"context"
+	"io/ioutil"
+	"os/exec"
+	"path/filepath"
+	"strconv"
+	"time"
+
+	_ "github.com/jinzhu/gorm/dialects/postgres"
```

Comment:
?

Reply:
fixed

Reply:
@avishayt please notice that i am not changing the local_job.go file. I have opened a ticket just for that. Once this commit will be pushed we can change the local_job, with the aid of disconnected team, since i have no way to check it. In any case they are still using the old code, so our commit should no break their code